config/test.exs

```diff
@@ -39,7 +40,7 @@ config :datagouvfr,
   community_resources_impl: Datagouvfr.Client.CommunityResources.Mock
 
 # capture all info logs and up during tests
-config :logger, level: :info
+config :logger, level: :debug
```

Comment:
Le code semble venir de https://elixirforum.com/t/exunit-capturelog-assert-capture-log-2-not-capturing-level-info/8617/10, je me demande pourquoi c'était mis à `:info`

Reply:
Avant c'était à warning, je l'avais baissé à info pour avoir accès aux logs d'infos et pouvoir tester que certains logs apparaissaient bien (on rajoutait des tests sur l'import, alors on ne voulait pas toucher au code existant, juste le tester).

Ca m'avait permis de faire ce [test](https://github.com/etalab/transport-site/pull/1617/commits/ac490b04b9b4ed67a9d2058cf25626e6c29363c7#diff-06c8c8a85ca2e8682be952aa870d243812d5a616149e3035360cab3638f0adf6R80).

Debug ça va capturer encore plus de choses.